CondCore/ESSources/plugins/CondDBESSource.cc

```diff
@@ -323,7 +325,37 @@ void CondDBESSource::fillList(const std::string& stringList,
 }
 
 CondDBESSource::~CondDBESSource() {
-  //dump info FIXME: find a more suitable place...
+  if (m_doDump) {
+    edm::Service<edm::JobReport> reportSvc;
+    ProxyMap::const_iterator b = m_proxies.begin();
+    ProxyMap::const_iterator e = m_proxies.end();
+    for (; b != e; b++) {
+      std::map<std::string, std::string> proxy_info;
+      const std::string& recName = (*b).first;
+      const auto& proxy = *(*b).second;
+      proxy_info["label"] = proxy.label();
+      proxy_info["connString"] = proxy.connString();
+      proxy_info["tag"] = proxy.tag();
+      const auto& pids = *proxy.requests();
+      for (const auto& id : pids) {
+        proxy_info[id.payloadId] = std::to_string(id.since) + " - " + std::to_string(id.till);
+      }
+      //reportSvc->reportAnalysisFile(recName, proxy_info);
+      reportSvc->reportPerformanceSummary(recName, proxy_info);
+    }
+
+    std::map<std::string, std::string> db_info;
+    db_info["DataProxy"] = std::to_string(m_stats.nData);
+    db_info["SetInterval"] = std::to_string(m_stats.nSet);
+    db_info["Runs"] = std::to_string(m_stats.nRun);
+    db_info["Lumis"] = std::to_string(m_stats.nLumi);
+    db_info["Refresh"] = std::to_string(m_stats.nRefresh);
+    db_info["ActualRefresh"] = std::to_string(m_stats.nActualRefresh);
+    db_info["Reconnect"] = std::to_string(m_stats.nReconnect);
+    db_info["ActualReconnect"] = std::to_string(m_stats.nActualReconnect);
+    reportSvc->reportPerformanceSummary("CondDBESSourceStats", db_info);
```

Comment:
Hi @vlimant!
Sorry for the maybe naive question, but wouldn't it be enough to change the `cout`s to the recommended `edm::LogInfo` or `edm::LogDebug`?

Reply:
Hello @francescobrivio , that is an alternative, though it still requires to parse the log in a non-trivial way ; not like the xml of job report.
any other way would be good for me though

Reply:
but `edm::LogPrint()` would not require any special parsing, right?

Reply:
LogPrint or LogDebug would still require a bunch of grep and else to be able to put this into usable form.
an alternative would be to write out a json directly from CondDBESSource when a dump_stats is requested.
I don't really understand why not using an existing vessel of framework information